samples/csharp/common/AutoML/ConsoleHelper.cs

```diff
@@ -109,6 +123,11 @@ internal static void PrintRegressionMetricsHeader()
             CreateRow($"{"",-4} {"Trainer",-35} {"RSquared",8} {"Absolute-loss",13} {"Squared-loss",12} {"RMS-loss",8} {"Duration",9}", Width);
         }
 
+        internal static void PrintRankingMetricsHeader()
+        {
+            CreateRow($"{"",-4} {"Trainer",-14}, {"nDCG@1",9}, {"nDCG@3",9}, {"nDCG@10",9}, {"DCG@10",9}, {"Duration",9}", Width);
```

Comment:
Extra commas:
```suggestion
            CreateRow($"{"",-4} {"Trainer",-14} {"nDCG@1",9} {"nDCG@3",9} {"nDCG@10",9} {"DCG@10",9} {"Duration",9}", Width);
```

Current prints as:
![image](https://user-images.githubusercontent.com/4080826/102622420-d8005e00-40f5-11eb-83d7-983f9b14350b.png)

Reply:
#Resolved